Runtime/Game/Requests/LeaderboardDetailsRequest.cs

```diff
@@ -266,7 +266,7 @@ public class LootLockerLeaderboardReward
         /// <summary>
         /// The Predicates of the reward.
         /// </summary>
-        public LootLockerLeaderboardRewardPredicates predicates { get; set; }
+        public LootLockerLeaderboardRewardPredicates[] predicates { get; set; }
```

Comment:
I copied the Details struct to LeaderboardRequests.cs. Please fix there as well 👌

Reply:
We've found that this was already done in LeaderboardRequests.cs, 🎉